front/app/containers/ProjectsShowPage/phaseParam.ts

```diff
@@ -0,0 +1,16 @@
+import { IPhaseData } from 'services/phases';
+
+// https://stackoverflow.com/a/10834843
+function isIntegerOverZero(str: string | undefined) {
+  if (str === undefined) return false;
+  const n = Math.floor(Number(str));
+  return n !== Infinity && String(n) === str && n > 0;
+}
+
+function phaseExists(phaseParam: string, phases: IPhaseData[]) {
+  return +phaseParam <= phases.length;
```

Comment:
I would watch out with the +conversion here, because what would happen if phaseParam is true? It would convert to 1.

Reply:
I think in this particular case it does not really matter, because `+''` returns `0`, meaning it would not get past the `isIntegerOverZero` test